src/url_request_handler.cc

```diff
@@ -70,11 +70,11 @@ void URLRequestHandler::getInfo(const char* filename, UpnpFileInfo* info)
         UpnpFileInfo_set_FileLength(info, -1);
     } else {
 #ifdef ONLINE_SERVICES
-        std::string url = item->getFlag(OBJECT_FLAG_ONLINE_SERVICE) ? OnlineServiceHelper::resolveURL(item) : item->getLocation().string();
+        fs::path url = item->getFlag(OBJECT_FLAG_ONLINE_SERVICE) ? OnlineServiceHelper::resolveURL(item) : item->getLocation();
```

Comment:
It's a URL so it starts with `http(s)://`. We should not declare that as path.

Reply:
What's interesting is getLocation() returns an fs::path

Reply:
It's the special situation of external items which inherit from CdsObject we have to live with, but we should make it an URL here (is there some additional lib for that?).